vllm_spyre/v1/worker/spyre_worker.py

```diff
@@ -60,6 +62,8 @@ def new_request_data_builder(
     }
 
     ## Temporary backwards compatibility for 0.10.2
+    # NOTE: currently multimodal only works on 0.11.0+
+    # If this is expected to be around a while, add bc
```

Comment:
If MM is only 0.11.0+ then we can rip out 0.10.2, we're not planning on any long-term support for older vllm versions atm. (We can do this in a separate PR)